### DISCUSSION TITLE
Student Verification expired but can't renew

### DISCUSSION THREAD

**Discussion by terryzfeng** [community/community#83533] [GitHub Education]
### Hi everyone,

My Github education student verification recently expired so I've been trying to renew it with my email. When selecting a school and hitting verify, I get a response bug:

```
Hi, terry feng! You were last verified as a student on . It is not necessary for you to re verify at this time.
```

I currently have no way to even upload documentation to reverify for Github education. How can this be fixed?

**Comment by samus-aran** [comment]
Hi @terryzfeng 👋 

We're sorry you're experiencing this. Can we please ask you to try now and re-apply. We've raised this experience with our internal team but we hope this helps you proceed to re-verify your academic status. 👍

**Reply by AnfengXu136** [reply]
I am having the same issue and it seems like the issue is still there. Could you please help?

**Reply by Miniland1333** [reply]
I am also still experiencing this issue.

**Reply by pengzhi1998** [reply]
I'm also experiencing the same issue. May I have your suggestions?

**Reply by AnselBlume** [reply]
I am also experiencing this issue.

**Reply by Miniland1333** [reply]
FYI the portal started working for me again. I do not know if they updated the code or if simply a month has passed since my previous verification expired.
![image](https://github.com/community/community/assets/12994749/2755db77-1a19-41ea-acd8-4f62cfdfd2ca)

**Reply by AnselBlume** [reply]
> FYI the portal started working for me again. I do not know if they updated the code or if simply a month has passed since my previous verification expired. ![image](https://private-user-images.githubusercontent.com/12994749/300965760-2755db77-1a19-41ea-acd8-4f62cfdfd2ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY2NTE5MzEsIm5iZiI6MTcwNjY1MTYzMSwicGF0aCI6Ii8xMjk5NDc0OS8zMDA5NjU3NjAtMjc1NWRiNzctMWExOS00MWVhLWFjZDgtNGY2MmNmZGZkMmNhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTMwVDIxNTM1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMzYzM0MzFlOWViZDc1YjYyYzgzNWFjZjAwOTkyNTlhNDhlMGJhOTMxMTMyZDgyYTFhZmZhYjM5ODJkNTQ5ODQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uNYiybNzgSXdJ7PH295y_KvWXeSxzwehFY32rsDu3Zc)

Thanks for the heads-up! It seems to be working for me now as well :)

**Reply by Miniland1333** [reply]
@pengzhi1998 did the portal start working for you as well? If so, then this issue is likely resolved!

**Comment by terryzfeng** [comment]
I was directly sent an email from Github to reverify my education which made it work but looks like this is still an ongoing fix. Just wanted to update my question and hope others are able to get theirs resolved too.

**Reply by pengzhi1998** [reply]
Thank you so much for your update!! May I ask which email you sent it to?

**Reply by terryzfeng** [reply]
Updated my comment, hopefully the Github team sees this thread and can email you too!

**Reply by pengzhi1998** [reply]
Got it. Thank you!!!
